docs/integrations/app-integrations/intercom.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Intercom
+sidebarTitle: Intercom
+---
+
+[Intercom](https://intercom.com) is a software company that provides customer messaging and engagement tools for businesses. They offer products and services for customer support, marketing, and sales, allowing companies to communicate with their customers through various channels like chat, email, and more.
+
+## Connection
+
+To get started with the Intercom API, you need to initialize the API handler with the required access token for authentication. You can do this as follows:
+
+- `access_token`: Your Intercom access token for authentication.
+
+## Usage
+
+To create a database using the Intercom engine, you can use a SQL-like syntax as shown below:
+
+```sql
+CREATE DATABASE myintercom
+WITH
+  ENGINE = 'intercom',
+  PARAMETERS = {
+    "access_token" : "your-intercom-access-token"
+  };
+```
```

Comment:
Please move the below content to the end of the `## Connection` section.
```
To create a database using the Intercom engine, you can use a SQL-like syntax as shown below:

```sql
CREATE DATABASE myintercom
WITH
  ENGINE = 'intercom',
  PARAMETERS = {
    "access_token" : "your-intercom-access-token"
  };
```
```